cognite/powerops/utils/identifiers.py

```diff
@@ -11,7 +11,7 @@ def unique_short_str(nbytes: int) -> str:
     return secrets.token_hex(nbytes=nbytes)
 
 
-def new_external_id(prefix: str = "POWEROPS_SHOP_RUN", now: Optional[datetime.datetime] = None) -> str:
+def new_external_id(prefix: str = "", now: Optional[datetime.datetime] = None) -> str:
```

Comment:
**question**: Do we need to have a default value for prefix?

Reply:
imo, no we do not -- and later maybe we should make it do that it is required argument